### DISCUSSION TITLE
Django-tailwind NPM_BIN_PATH location node

### DISCUSSION THREAD

**Discussion by adammakowskidev** [cookiecutter/cookiecutter-django#3404] [Q&A] [answered]
Hi
Im trying to use django-tailwind
https://github.com/timonweb/django-tailwind
I use Django Cookiecutter with node docker image
I have a problem with the location NPM_BIN_PATH
Does anyone know how to point to the correct location for node docker image?

```

CommandError: 
It looks like node.js and/or npm is not installed or cannot be found.

Visit https://nodejs.org to download and install node.js for your system.

If you have npm installed and still getting this error message, set NPM_BIN_PATH variable in settings.py to match path of NPM executable in your system.

Example:
NPM_BIN_PATH = "/usr/local/bin/npm"

```

**Comment by foarsitter** [comment, marked as answer]
This is the image of your local docker Django container: https://github.com/cookiecutter/cookiecutter-django/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/compose/local/django/Dockerfile
Do you notice the word `node` does not occur at all?  Thats right, the node container is responsible for the processing of css and js and is completely separated from the Django container.  Referencing npm in the Django container is thus not possible since it is not installed at all. 

You need to modify the Django Dockerfile and install node by yourself. This guide will help you (option 2 has my preference): https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-20-04 

Last but not least: if you need help, Github issues is not the best place to get it. Try stackoverflow for issues not related to this project.

**Comment by PositivPy** [comment]
I am attempting to integrate django-tailwind with this cookie cutter as well. There seems to be an issue with the docker file process, as I am encountering errors with DATABASE_URL, CELERY_BROKER_URL, USE_DOCKER, and NPM_BIN_PATH.
